### PR TITLE
feat: Add Women Who Code in the communities category

### LINKED ISSUE #1821: [Add] Women Who Code in the communities sub category
### Site URL

https://www.womenwhocode.com/communities/

### Description of the above provided site

Women Who Code is a global non-profit organization dedicated to change the industry experience for women engineers. It is an active community that aims to provide an avenue into the technology world by evaluating and assisting women in developing technical skills.

### Category

Other

### Sub Category

Communities

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by SargamPuram**
## Fixes Issue

Closes #1821 

## Changes proposed
It adds Women Who Code in the communities category.

## Screenshots
wwc
wwc1

## Note to reviewers
Kindly add the hactoberfest accepted label.

### CODE REVIEW

File: database/other/communities.json [hunk 1]
```diff
@@ -54,5 +54,12 @@
     "url": "https://vexpert.vmware.com/",
     "category": "other",
     "subcategory": "communities"
+  },
+  {
+    "name": "Women Who Code",
+    "description": "Women Who Code is a global non-profit organization dedicated to change the industry experience for women engineers. It is an active community that aims to provide an avenue into the technology world by evaluating and assisting women in developing technical skills.",
```
Comment (CBID2) [inline database/other/communities.json:60]: [suggestion] I'm a member of one of their communities and they always strive to create inclusive environments for women and non-binary individuals from different backgrounds, so I recommend mentioning this point as shown in the example below

```suggestion
    "description": "Women Who Code is a global non-profit organization dedicated to providing women and non-binary individuals from diverse backgrounds with the necessary tools and skills to thrive in the tech industry.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @SargamPuram! :) I just gave some feedback on your description. Other than that, you're off to a great start! 😃

File: database/other/communities.json (hunk 1 above)
Comment (SargamPuram) [reply database/other/communities.json]: @CBID2 thanks for mentioning it , just made the changes  in the description. Please have a look .

**Comment by SargamPuram** [comment]
Hey, @CBID2 I've made the changes, please can you have a look.

**Review by CBID2 — APPROVED** [review]
Well done @SargamPuram! 😃 @Anmol-Baranwal, you're up! 😀

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**